lib/Client.js

```diff
@@ -478,6 +478,49 @@ class Client extends EventEmitter {
         return this.requestHandler.request("POST", Endpoints.CHANNEL_WEBHOOKS(channelID), true, options);
     }
 
+    /**
+    * Create a global slash command
+    * @arg {Object} command Command object
+    * @arg {String} command.name The command name
+    * @arg {String} command.description The command description
+    * @arg {String} [command.options] The parameters for the command
+    * @arg {Boolean} [defaultPermission] Whether the command is enabled by default when the app is added to a guild
+    * @returns {Promise<Object>} Resolves with a commands object
+    */
+    createCommand(command) {
+        if(!command.name || !command.description) {
+            throw new Error("You must define a command name and description.");
+        }
+        command.name = command.name.toLowerCase();
+        if(!command.name.match(/^[\w-]{1,32}$/)) {
+            throw new Error("Command names must match the regular expression \"^[\\w-]{1,32}$\"");
+        }
+        return this.requestHandler.request("POST", Endpoints.COMMANDS(this.application.id), true, command);
+    }
+
+    /**
+    * Bulk create global slash commands
```

Comment:
isn't this should be Bulk update or edit global slash commands as in discord docs
https://discord.dev/interactions/slash-commands#edit-guild-application-command

Reply:
It is, and you linked the wrong docs section
https://discord.com/developers/docs/interactions/slash-commands#bulk-overwrite-global-application-commands
https://github.com/Catboi8/eris/blob/slash-commands/lib/rest/Endpoints.js#L10

Reply:
@DonovanDMC my scroller bad, but yeah I don't think createCommands is right.

Reply:
Yes in the docs its bulk update but i went the way of deleteMessages and just made it createCommands, tho maybe i should make it createGlobalCommands, also i will change it if enough people think it should be

also not only bulk update uses the commands endpoint so that would stay the same

Reply:
Also it isnt only for editing commands is the thing, it creates and simply overrides existing commands if the names match

Reply:
bulkEditCommands sounds better 👍
because that endpoint updates existing commands too and it doesn't only create

Reply:
Alright 3 is enough for me, one min